src/components/common/madie-link.scss

```diff
@@ -0,0 +1,13 @@
+.madie-link {
+  align-self: center;
+  height: 16.52px;
+  font-family: "Rubik", sans-serif;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 17px;
+  text-align: center;
+}
+.add-icon {
```

Comment:
May not be an ideal thing to do right now, but I was thinking since these Styles have to be used for every newly added link in madie, can we add them to our component in design-system ?

Reply:
@RohitKandimalla @adongare Thanks a lot Rohit and Ashok. I thought about moving all styles to design-system, but since this is about the unified color change for the link, and also for text alignment, padding, etc, it might be more component specific, and that's why I didn't move everything to design-system. I think it's a balance thing.